conf/messages.en

```diff
@@ -119,14 +119,22 @@ distribution.error.outOfRange = Distributions must be between {0} and {1}
 distribution.change.hidden = Distributions
 
 associatedCompanies.title = Number of associated companies
-associatedCompanies.heading = Number of associated companies
-associatedCompanies.checkYourAnswersLabel = Number of associated companies
-associatedCompanies.error.nonNumeric = Enter your Number of associated companies using numbers
-associatedCompanies.error.required = Enter your Number of associated companies
-associatedCompanies.error.wholeNumber = Enter your Number of associated companies using whole numbers
-associatedCompanies.error.outOfRange = Number of associated companies must be between {0} and {1}
-associatedCompanies.change.hidden = Number of associated companies
-
+associatedCompanies.heading = Did your company have any active associated companies between 1 April 2023 to 31 December 2023?
+associatedCompanies.warning = Do not include the company you're currently calculating Marginal Relief for in your answer. Doing so will provide you with an incorrect calculation.
+associatedCompanies.detailsTitle = What are associated companies?
+associatedCompanies.detailsHtml = A company is considered associated to another if:<br/><ul><li>one of the companies has <a href="https://www.gov.uk/hmrc-internal-manuals/company-taxation-manual/ctm60210" class="govuk-link">control</a> of the other</li><li>both companies are controlled by the same companies or people</li>
+associatedCompanies.option1Value = yes
```

Comment:
value should not be in this file, else our backend will fail to work :)

Reply:
Moved option values and text inline in the view